src/Symfony/Bridge/Doctrine/ArgumentResolver/EntityValueResolverVariableInjector.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bridge\Doctrine\ArgumentResolver;
+
+use Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface;
+
+/**
+ * Default variable loader service used to inject user in EntityValueResolver
+ *
+ * @author Roman JOLY <eltharin18@outlook.fr>
+ */
+class EntityValueResolverVariableInjector implements EntityValueResolverVariableInjectorInterface
+{
+    public function __construct(private readonly TokenStorageInterface $tokenStorage)
```

Comment:
I'm not 100% sure, but _I think_ there was a discussion in this repo lately to remove `readonly`  in the case of services because it's just adding noise without much added value. Readonly properties are generally more suitable to DTO and VOs

Reply:
no problem to supress it